docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -333,6 +333,24 @@ nQIDAQAB
 -----END PUBLIC KEY-----
 ----
 
+=== Generating Keys
+
+It is also possible to generate a public and private key pair using the OpenSSL command line tool.
+
+.openssl commands for generating keys
+[source, text]
+----
+openssl genrsa -out rsaPrivateKey.pem 4096
```

Comment:
Please use `2048` as the keys used in the demo are `2048` bit length AFAIK as well (if you'd like, please add a comment that one can increase this value to `4096`/etc)

Reply:
Oops, I didn't notice that.  I'll update it.